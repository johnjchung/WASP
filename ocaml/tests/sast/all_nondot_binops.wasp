Endpoint allBinops(GET:) {
	
	int a = 3 + 4;
	int b = 1 + 3.5;
	int c = 3.4 + 2.4;
	int d = 3.5 + 1;
	string e = "test" + " works";
	
	int f = 2 - 5;
	int g = 4 - 1.4;
	int h = 2.5 - 2.2;
	int i = 2.6 - 1;
	
	int j = 1 * 5;
	int k = 4 * 1.3;
	int l = 1.5 * 14.5;
	int m = 1.6 * 1;

	int n = 5 / 2;
	int o = 10 / 2.5;
	int p = 6.6 / 3.1;
	int q = 5.5 / 2;
}
