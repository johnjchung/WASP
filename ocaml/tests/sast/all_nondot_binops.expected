Endpoint allBinops( GET:) {int a = [int] (Binop ([int] 3) + ([int] 4));, int b = [int] (Binop ([int] 1) + ([float] 3.5));, int c = [int] (Binop ([float] 3.4) + ([float] 2.4));, int d = [int] (Binop ([float] 3.5) + ([int] 1));, string e = [string] (Binop ([string] "test") + ([string] " works"));, int f = [int] (Binop ([int] 2) - ([int] 5));, int g = [int] (Binop ([int] 4) - ([float] 1.4));, int h = [int] (Binop ([float] 2.5) - ([float] 2.2));, int i = [int] (Binop ([float] 2.6) - ([int] 1));, int j = [int] (Binop ([int] 1) * ([int] 5));, int k = [int] (Binop ([int] 4) * ([float] 1.3));, int l = [int] (Binop ([float] 1.5) * ([float] 14.5));, int m = [int] (Binop ([float] 1.6) * ([int] 1));, int n = [int] (Binop ([int] 5) / ([int] 2));, int o = [int] (Binop ([int] 10) / ([float] 2.5));, int p = [int] (Binop ([float] 6.6) / ([float] 3.1));, int q = [int] (Binop ([float] 5.5) / ([int] 2));}
